test/example/app/controllers/application_controller.rb

```diff
@@ -1,8 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
 
-  skip_before_filter :authenticate, :only => :unauthenticated
```

Comment:
I have no idea where this came from, but it was breaking because `:authenticate` doesn't exist